src/documentation/ToolkitDocumentation/ToolkitContent.tsx

```diff
@@ -75,8 +75,10 @@ const ToolkitInternalLinkMark = (
         e.preventDefault();
         setState({
           ...state,
-          // Hmm, we need to know the tab/toolkit (we should name them the same).
-          // We also need to switch to router-based navigation for the other toolkits.
+          tab: "explore",
+          explore: {
+            id: props.mark.slug.current,
```

Comment:
We need to conditionally use the slug as it is if `targetType === "toolkitTopic"`. Else, we need to somehow get the parent topic slug and join this with the entry slug.

Reply:
Resolved with commit 87dbf74.